web/client/components/map/leaflet/Feature.jsx

```diff
@@ -71,13 +71,13 @@ const geometryToLayer = function(geojson, options) {
 
     case 'LineString':
         latlngs = coordsToLatLngs(coords, geometry.type === 'LineString' ? 0 : 1, coordsToLatLng);
-        layer = new L.Polyline(latlngs, options.style);
+        layer = new L.Polyline(latlngs, options.style && options.style.LineString || options.style);
```

Comment:
We should use the type to select the style (options.style && options.style[type] || options.style), and possibly move it outside the switch:

```
const style = options.style && options.style[type] || options.style;
....
layer = new ....(..., style);
```